docs/abi-spec.rst

```diff
@@ -412,7 +412,10 @@ A function description is a JSON object with the fields:
 - ``outputs``: an array of objects similar to ``inputs``, can be omitted if function doesn't return anything;
 - ``payable``: ``true`` if function accepts ether, defaults to ``false``;
 - ``stateMutability``: a string with one of the following values: ``pure`` (:ref:`specified to not read blockchain state <pure-functions>`), ``view`` (:ref:`specified to not modify the blockchain state <view-functions>`), ``nonpayable`` and ``payable`` (same as ``payable`` above).
-- ``constant``: ``true`` if function is either ``pure`` or ``view``
+
+.. warning::
+    Prior to version 0.5.0 there was an additional field ``constant``. In 0.5.0 this field has been removed.
+    Instead you can check whether ``stateMutability`` is ``pure`` or ``view``.
```

Comment:
We should create a PR adding a deprecation warning for both `constant` and `payable`.

Reply:
Did that in #4549.